docs_user/modules/openstack-mariadb_copy.adoc

```diff
@@ -55,16 +50,77 @@ endif::[]
 ifeval::["{build}" == "downstream"]
 MARIADB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-mariadb-rhel9:18.0
 endif::[]
+STORAGE_CLASS=crc-csi-hostpath-provisioner
 # Replace with your environment's MariaDB Galera cluster VIP and backend IPs:
 SOURCE_MARIADB_IP=192.168.122.99
 declare -A SOURCE_GALERA_MEMBERS
-SOURCE_GALERA_MEMBERS=(
+SOURCE_GALERA_MEMBERS=
   ["standalone.localdomain"]=192.168.122.100
   # ...
 )
 SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
 ----
 
+* Prepare MariaDB copy directory and the adoption helper pod
+
+* Create a temporary folder to store adoption helper pod (pick storage requests to fit MySQL database size):
++
+----
+mkdir ~/adoption-db
+cd ~/adoption-db
+----
+[source,yaml]
+----
+oc apply -f - <<EOF
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mariadb-data
+spec:
+  storageClassName: $STORAGE_CLASS
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mariadb-copy-data
+  annotations:
+    openshift.io/scc: anyuid
```

Comment:
please add ```k8s.v1.cni.cncf.io/networks: internalapi```, which probably becomes```k8s.v1.cni.cncf.io/networks: internal_api``` after https://github.com/openstack-k8s-operators/install_yamls/pull/758

Reply:
@holser I think we need this, please see the same for ovn data copy tasks

Reply:
nvm, agreed with Harald